app/src/main/java/space/taran/arknavigator/utils/Tags.kt

```diff
@@ -20,7 +20,7 @@ class Converters {
             .split(',')
             .map { it.trim() }
             .filter {
-                if (it.isEmpty()) throw AssertionError("No tag can be empty")
+                if (it.isEmpty()) return@filter false
```

Comment:
In this case, yes, sometimes we need to suppress the assertion, but we don't want allow messy tags strings being read from tags storage file. Can we fix the problem without removing this assertion?

Reply:
Actually, just remove this please. #101 will be fixed by #117.